public/js/reader.js

```diff
@@ -118,7 +125,9 @@ Reader.handleShortcuts = function (e) {
         break;
     case 32: // spacebar
         if ($(".page-overlay").is(":visible")) { break; }
-        if ((window.innerHeight + window.scrollY) >= document.body.offsetHeight) {
+        if (e.originalEvent.getModifierState("Shift") && (window.scrollY) === 0) {
+            (Reader.mangaMode) ? Reader.changePage(1) : Reader.changePage(-1);
```

Comment:
No, the scrollbar logic is different from the arrow keys. While D is forwards in normal mode and backwards in manga mode, spacebar is always forwards in both modes, so it needs to be reversed accordingly to always result in a positive offset.

Reply:
Right, forgot about this despite asking for it to be added in the previous PR 😅 This might be worth adding a comment to document the behavior as well.

Reply:
Done, added a quick comment for future generations.